CODECONVENTIONS.md

```diff
@@ -51,11 +51,32 @@ integer types.  The general guidelines are:
 - You can use int/uint, but remember that they may be 16-bits wide.
 - If in doubt, use mp_int_t/mp_uint_t.
 
+Documentation:
+- Be concise
+- Only write comments for things that are not obvious
+- In general, there should not be lengthy or unnecessary comments. If something
+  may be confusing to new contributors or people unfamiliar with computer
+  science, it is fine to use a few key words and a link to a wikipedia article
+  or a wiki page.
```

Comment:
I hate to be picky on such a trivial thing, but the philosophy of what you've written here goes against itself: each point is an embellishment of the previous one!  It would be enough to say "be concise".  If you want to say more than that then "be concise and anly write comments for things that are not obvious" would be pretty good.